tests/kola/butane/grub-users/test.sh

```diff
@@ -36,18 +36,18 @@ rebooted)
         fatal "Rebooted into old deployment"
     fi
     # cross-check karg with BLS configs
-    if grep -q test-added-karg /boot/loader.0/entries/ostree-1-fedora-coreos.conf; then
+    if grep -q test-added-karg /boot/loader.0/entries/ostree-1.conf; then
```

Comment:
We need the test to work with the old filename too. Maybe simplest is just e.g.

```suggestion
    if grep -q test-added-karg /boot/loader.0/entries/ostree-1*.conf; then
```